src/js/components/Drop/DropContainer.js

```diff
@@ -15,6 +15,30 @@ import { StyledDrop } from './StyledDrop';
 import { OptionsContext } from '../../contexts/OptionsContext';
 import { useThemeValue } from '../../utils/useThemeValue';
 
+// Check if the element.tagName is an input, select or textarea
+export const isFocusableElements = (element) => {
```

Comment:
Do we need to export this? Looks like we are only referencing it in this file

Reply:
yes your correct